sim_pipeline/galaxy_galaxy_lens_pop.py

```diff
@@ -132,29 +127,29 @@ def select_lens_at_random(self, **kwargs_lens_cut):
             if gg_lens.validity_test(**kwargs_lens_cut):
                 return gg_lens
 
-    def deflector_number(self):
+    def potential_deflector_number(self):
         """Number of potential deflectors (meaning all objects with mass that are being
         considered to have potential sources behind them)
 
         :return: number of potential deflectors
         """
-        return self._lens_galaxies.deflector_number()
+        return len(self._lens_galaxies)
 
-    def source_number(self):
+    def potential_source_number(self):
         """Number of sources that are being considered to be placed in the sky area
         potentially aligned behind deflectors.
 
-        :return: number of sources
+        :return: number of potential sources
         """
-        return self._sources.source_number()
+        return len(self._sources)
```

Comment:
same as from the deflector numbers. It can be ok to duplicate access points in different classes as long as the underlying function is the same to be called

Reply:
Okay, this makes sense! In galaxy_galaxy_lens_pop, I renamed deflector_number() to potential_deflector_number() and source_number() to potential_source_number(). Do you recommend that I keep this name and use the same method name (i.e., potential_deflector_number and potential_source_number) for Deflector and Source objects instead of len(...)?

Reply:
hmm, I am a bit torn here. 'potential' is also used for lensing potential in certain terms. it might be easier to keep them without 'potential', or 'all' or 'unlensed'. I leave it up to @nkhadka21 and you (@padma18-vb ) :)

Reply:
I prefer to keep them without potential. It is also easier to keep track of test functions with fewer changes.

Reply:
Yes, those are all valid points, will revert both methods to source_number()!